src/goto-cc/Makefile

```diff
@@ -51,6 +46,8 @@ all: goto-cl$(EXEEXT)
 endif
 all: goto-cc$(EXEEXT)
 
+as_mode$(OBJEXT) compile$(OBJEXT) gcc_mode$(OBJEXT) goto_cc_mode$(OBJEXT): $(GIT_INFO_FILE)
```

Comment:
Ditto my comment about whether the dependency would now be auto-generated from the `.d` files? (and all the other */Makefile's)